datagen/procthor_datagen/datagen_runner_valid.py

```diff
@@ -0,0 +1,1362 @@
+"""A script for generating rearrangement datasets."""
```

Comment:
Can we merge datagen_runner_valid.py and datagen_runner_train.py? From what I can see they're essentially identical?

Reply:
👍